tests/integration.rs

```diff
@@ -90,3 +104,42 @@ fn integration_test() {
         None => panic!("Process terminated by signal"),
     }
 }
+
+#[cfg_attr(feature = "integration", test)]
+fn test_sysroot() {
+    let rustc = std::env::var("RUSTC").unwrap_or("rustc".to_string());
+    let rustc_output = Command::new(rustc)
+        .args(["--print", "sysroot"])
+        .output()
+        .expect("unable to run rustc");
+    assert!(rustc_output.status.success());
+    let sysroot = String::from_utf8(rustc_output.stdout).unwrap();
+    let sysroot = sysroot.trim_end();
+
+    #[track_caller]
+    fn verify_cmd(cmd: &mut Command) {
```

Comment:
Making this a separate function isn't strictly necessary but it will make it easier to test clippy-driver in the future if you choose to.